### PR TITLE
fix(DB_Creature) remove unnecessary NPC Griz Gutshank

### PR DESCRIPTION
**Pull request by JamesGent**
## Changes Proposed:
-   remove unnecessary NPC Griz Gutshank

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7676

## SOURCE:

https://www.wowhead.com/npc=28225/griz-gutshank#comments

By symet on 2008/11/12 (Patch 3.0.3) |   |  
He seems to have disappeared from Blackrock Depths, post arena season 4 ending.

## Tests Performed:

- No tests perform, SQL only fix.
- 

## How to Test the Changes:

1. Run SQL
2. Launch server
3.GO to Grim Guzzler/check for NPC

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Si1ker** [comment]
<html>
<body>


1 | sinizuh on 2008/12/08 (Patch 3.0.3)I just ran BRD and I can confirm he's still active.
-- | --



</body>
</html>


<html>
<body>


0 | caskeper on 2009/07/18 (Patch 3.1.3)Just ran BRD and he's (still) there (again).
-- | --



</body>
</html>



image

interesting.

File: data/sql/updates/pending_db_world/rev_1630506569314815219.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1630506569314815219');
+
+DELETE FROM `creature` WHERE (`id` = 28225);
```
Comment (Asthalor) [inline data/sql/updates/pending_db_world/rev_1630506569314815219.sql:3]: Add a new line so the build wont fail :P

**Comment by JamesGent** [comment]
> 
> 1 	sinizuh on 2008/12/08 (Patch 3.0.3)I just ran BRD and I can confirm he's still active.
> 0 	caskeper on 2009/07/18 (Patch 3.1.3)Just ran BRD and he's (still) there (again).
> 
> image
> 
> interesting.

The sources of information that he was removed are SYMET, a 2000+ reputation premium wowhead user with thousands of accurate comments and DrCheis, a 50,000+ reputation premium wowhead user with hundreds of thousands of accurate comments/notes/posts. The comment saying he was removed is at +5, meaning 5 other users agreed that he was gone.
The comments saying he's "still there" are at +1/-0 meaning people didn't agree.

He was added in 2.4.3, for the October 2008 TBC Brewfest event. WOTLK launched on November 2008. 3.3.5 is based on 22 June 2010, a full 2 years after Griz became redundant/pointless and people discussed his removal.

Since we don't know when exactly he was removed - somewhere between 3.0.1 and 3.1.3, we can safely assume he probably was removed in the end by 3.3.5a which we're based on as AzerothCore.

It's a shame we don't have a clean cut perfect source, but it's good enough for me to clean up a useless NPC.

Desktop 01-09-2021 23-08-54-671

File: data/sql/updates/pending_db_world/rev_1630506569314815219.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1630506569314815219');
+
+DELETE FROM `creature` WHERE `id` = 28225;
+
+
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1630506569314815219.sql:5]:
```suggestion
DELETE FROM `creature` WHERE `id` = 28225;

```

**Review by Si1ker — APPROVED** [review]

**Comment by Si1ker** [comment]
```txt
Run for i in `find data/sql/updates/pending* -name "*.sql" -type f`; do
Missing Semicolon (;) or multiple newlines at the end of the file.
Error: Process completed with exit code 1.
```

I don't get it 😕 ❓

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan what's happening here with the build?

**Comment by r-o-b-o-t-o** [comment]
> @Winfidonarleyan what's happening here with the build?

It has windows line-endings which appear to break the verification script.
I sent a fixed version to @JamesGent on Discord in the #support-mysql channel.

**Comment by r-o-b-o-t-o** [comment]
@FrancescoBorzi fixed!

**Pull request merged**